functions/openhab.js

```diff
@@ -151,6 +168,59 @@ function getItemsState(request,response) {
 		})
 }
 
+/**
+ * Gets Temperature or Thermostat Data
+ */
+function getTempData(item) {
+	var thermData = {};
+	var thermItems = item.tags.toString().includes("Thermostat") ? getThermostatItems(item.members) : getThermostatItems([item]);
+	
+	//Are we dealing with Fahrenheit?
+	var isF = item.tags.indexOf('Fahrenheit') >= 0 ? true : false;
```

Comment:
1. use `let` or `const` in ES6
1. `const isF = item.tags.includes('Fahrenheit');`